package.json

```diff
@@ -25,11 +25,11 @@
   },
   "homepage": "https://github.com/devongovett/browserify-istanbul",
   "dependencies": {
-    "istanbul": "~0.3.6",
     "minimatch": "^2.0.0",
     "through": "^2.3.4"
   },
   "devDependencies": {
+    "istanbul": "^0.4.2",
```

Comment:
Yep, it's ideal case. Although current state of peerDependencies in npm (especially v2.x) is less than ideal.

When we all switch to npm@3 and it will polish out all the bugs,
we'd be able to enjoy peer-dependencies goodness.

Details could be found here:
- https://docs.npmjs.com/files/package.json#peerdependencies
- https://github.com/npm/npm/issues/6565